### PR TITLE
Add Stefan and Warren as authors for -04 draft

### PR DESCRIPTION
**Pull request by ekline**
@panmari @wkumari 

Before we publish a -04 draft we should get you guys onto the authors list.

(Also: update my affiliation, since this document has been around for a while. =)

### CODE REVIEW

File: draft.xml [hunk 1]
```diff
@@ -73,18 +64,46 @@
...
           <city>Zürich</city>
-
           <country>Switzerland</country>
         </postal>
 
         <email>zszami@google.com</email>
       </address>
     </author>
 
+    <author fullname="Stefan Moser" initials="S." surname="Moser">
```
Comment (panmari) [inline draft.xml:76]: Thanks for adding me! Please also add

Maciej Kuzniar

**Comment by ekline** [comment]
Have they contributed some text (vicariously)?  Generally there's a 5 author limit:

    https://www.rfc-editor.org/policy.html#policy.authlist

Though it's not technically a hard limit, it will raise this as a question as we go through the process.  I could add them to the Acknowledgements section?

**Comment by ekline** [comment]
Ah! Warren is in the Acknowledgements section.  I should fix that in this same change.

**Comment by ekline** [comment]
Closing this in favor of https://github.com/google/self-published-geo/pull/6

**Pull request closed without merging**